libs/input-mapping/tests/Functional/DownloadFilesAdaptiveBranchTest.php

```diff
@@ -93,17 +71,19 @@ public function testReadFilesAdaptiveWithBranch(): void
 
         self::assertTrue($this->testHandler->hasInfoThatContains(
             sprintf(
-                'Using dev tags "%s-%s, %s-adaptive" instead of "%s, adaptive".',
-                $branchId,
-                self::TEST_FILE_TAG_FOR_BRANCH,
-                $branchId,
-                self::TEST_FILE_TAG_FOR_BRANCH,
+                'Using dev tags "%s-%s, %s-%s-adaptive" instead of "%s, %s-adaptive".',
+                $this->devBranchId,
+                $this->testFileTagForBranch,
+                $this->devBranchId,
+                $this->testFileTagForBranch,
+                $this->testFileTagForBranch,
+                $this->testFileTagForBranch,
```

Comment:
🤯

Reply:
Trochu jsem to vylepsil https://github.com/keboola/platform-libraries/pull/352/commits/e73d8a18c78183f3b0b3b2d3e19b8347374a7111